src/components/People/TechStack.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+import klecks from '../Lakritzel/Assets/Kleckse_clean.jpg'
+
+function TechStack() {
+  return (
+    <Wrapper>
+      <Headline>TechStack</Headline>
+      <List>
+        <ListItem>HTML</ListItem>
+        <ListItem>CSS</ListItem>
+        <ListItem>React</ListItem>
+        <ListItem>JavaScript</ListItem>
+        <ListItem>Node.js</ListItem>
+        <ListItem>ExpressJS</ListItem>
+        <ListItem>MongoDB</ListItem>
+        <ListItem>Git & GitHub</ListItem>
+        <ListItem>Figma</ListItem>
+      </List>
```

Comment:
May be adding "React-Testing Library", "Storybook", "Vercel", "VS Code", "Mongoose"?
And instead of "HTML" and "CSS3" "HTML5" and "CSS3"?